doc/howto/write_a_view/index.md

```diff
@@ -0,0 +1,465 @@
+// a is an adaptor and can be used as such:
+auto v = vec | a;
+```
+
+  4. You can create a new *adaptor closure object* from existing adaptor closure objects via `|` but without providing
+  a range:
+```cpp
+std::vector vec{1, 2, 3, 4, 5};
+auto a = std::view::transform([] (int i) { return i + 1; })
+       | std::view::filter([] (int i) { return i % 2 == 0; });
+// a is an adaptor and can be used as such:
+auto v = vec | a;
+```
+
+| Terminology                    | Description                                                             | Example                                                                       |
+|--------------------------------|-------------------------------------------------------------------------|-------------------------------------------------------------------------------|
+| "view"                         | A type that models std::ranges::View.                                   | `std::ranges::filter_view<std::subrange<int const *, int const *>>`           |
+| "view adaptor object"          | Creates a view; can be combined with other adaptors.                    | `std::view::filter` and <br> `(std::view::filter([] (int) { return true; }))` |
+| "view adaptor closure object"  | A "view adaptor object" that takes no paramaters other than the range.  | `(std::view::filter([] (int) { return true; }))`                              |
+
+In many cases where you are planning on creating "a new view", it will be sufficient to use the previously
+mentioned techniques to just create "a new adaptor object" and not having to specify the actual view type yourself.
+
+Let's look at some examples!
+
+\assignment{Exercise 1: Your first custom adaptor object}
+In the alphabet module you learned that ranges of alphabets are not implicitly
+convertible to `char` so you **cannot** print a `std::vector<seqan3::dna5>` via `std::cout`¹.
+You also know that you can call `seqan3::to_char` on every object that models `seqan3::Alphabet` which will convert it
+to `char` or a similar type.
+
+We want to do the following:
+
+\snippet doc/howto/write_a_view/view_exercise1.cpp start
+```cpp
+// your implementation goes here
+```
+\snippet doc/howto/write_a_view/view_exercise1.cpp end
+
+Define a *range adaptor object* using an existing adaptor which applies a concrete transformation
+(calling seqan3::to_char) on every element.
+
+\hint
+You need to need use `std::view::transform` and you need to set a fixed transformation function. `std::view::transform`
+takes an object that models `std::RegularInvocable`, e.g. a lambda function with empty capture `[]`.
+\endhint
+
+<small>¹ You *can* print via `seqan3::debug_stream`, but let's ignore that for now. </small>
+\endassignment
+
+\solution
+
+\include doc/howto/write_a_view/view_exercise1.cpp
+
+You simply define your adaptor type as `auto` and make it behave like `std::view::transform`, except that you
+"hard-code" the lambda function that is applied to each element.
```

Comment:
Add: Since your adaptor object now only takes a range as the only parameter, it is an adaptor closure object.